contextual_lenses/pfam_family_classification/pfam_one_hot_lenses.py

```diff
@@ -3,7 +3,7 @@
 
 # Parse command line arguments.
 from parser import parse_args
-tpu_name, save_dir, restore_dir, use_pmap = parse_args()
+tpu_name, save_dir, restore_dir, use_pmap, restore_transformer_dir, bidirectional = parse_args()
```

Comment:
not necessary to change, but in the future it would be cleaner to have parse_args() return a collections.namedtuple. This way, when you add more possible command line args you don't have to ever place that parse_args is called. You also remove the number of global variables. 

lense_config = parse_args()
...
foo(lens_config.use_pmap)

Reply:
great idea, the growing list of command-line arguments was worrying me as well